docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_in_gcs.rst

```diff
@@ -1,18 +1,110 @@
 .. _how_to_guides__configuring_metadata_stores__how_to_configure_an_expectation_store_in_gcs:
 
-How to configure an Expectation store in GCS
```

Comment:
I know that we have GCS everywhere (including references in code), but Google seems to want to call it Google Cloud Platform (GCP). Should I keep things as they stand now? (as GCS)